torch/_torch_docs.py

```diff
@@ -445,6 +445,32 @@ def merge_dicts(*dicts):
             [ 0.7573, -3.9555, -2.8681]])
 """.format(**common_args, **tf32_notes))
 
+add_docstr(torch.sspaddmm,
+           r"""
+sspaddmm(input, mat1, mat2, *, beta=1, alpha=1, out=None) -> Tensor
+
+This function does exact same thing as :func:`torch.addmm` except the
+input and the output are sparse matrices.
+
+Args:
+    input (Tensor): a sparse matrix to be added
+    mat1 (Tensor): a sparse matrix to be multiplied
+    mat2 (Tensor): a dense matrix to be multiplied
+    beta (Number, optional): multiplier for :attr:`mat` (:math:`\beta`)
+    alpha (Number, optional): multiplier for :math:`mat1 @ mat2` (:math:`\alpha`)
+""")
+
+add_docstr(torch.smm,
+           r"""
+smm(input, mat) -> Tensor
+
+This function computes ``input @ mat`` with sparse tensor as output.
+
+Args:
+    input (Tensor): a sparse matrix to be added
```

Comment:
"the sparse matrix to be matrix multiplied" (not added)

Reply:
fixed.